roles/confluent.test/molecule/archive-community-plaintext-rhel/molecule.yml

```diff
@@ -99,9 +99,6 @@ provisioner:
 
 verifier:
   name: ansible
-lint: |
```

Comment:
I'd like to see this GH action "in action" before removing from molecule. Can we make this PR just turning on gh actions for now. IE have both for now

Reply:
NVRM, this would be redundant. I think the .yamllint file should be removed then as well

Reply:
But why remove `.yamllint`? It is configuring the yamllint done by the GH workflow, and allows developers/contributors to run yamllint locally.

Reply:
nvrm x2